### PR TITLE
fix: fix accuracy in decentralized flow

### PR DESCRIPTION
**Pull request by Adibov**

### CODE REVIEW

File: app/entity/decentralized_client.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def initialize(self, split_layer, LR):
         self.split_layers = split_layer
 
         fed_logger.debug('Building Model.')
-        self.net = model_utils.get_model('Client', self.split_layers, self.device, self.edge_based)
+        # self.net = model_utils.get_model('Client', self.split_layers, self.device, self.edge_based)
```
Comment (Adibov) [inline app/entity/decentralized_client.py]: Why we had this?

File: app/fl_training/flow/fed_edgeserver_flow.py [hunk 2]
```diff
@@ -112,28 +111,27 @@ def run_decentralized_offload(server: FedDecentralizedEdgeServer, learning_rate,
...
+        server.split(neighbors_bandwidth_by_type[NodeType.CLIENT], options)
         fed_logger.info(f"Split Config : {server.split_layers}")
         server.scatter_split_layers()
 
-        if r > 49:
-            learning_rate = config.LR * 0.1
-
         fed_logger.info("initializing server")
-        server.initialize(learning_rate)
+        # server.initialize(learning_rate * (0.1 ** (r // 10)))
```
Comment (Adibov) [inline app/fl_training/flow/fed_edgeserver_flow.py]: I'm also ignoring calling the initialize method. Check if it's the desired behavior.
